app/controllers/shops_controller.rb

```diff
@@ -29,7 +29,7 @@ def edit
 
  	def update
     if @shop.update(shop_params)
```

Comment:
Indentation is messy :/ Are you using consistent indentation? It's all messed up on mine...